### PR TITLE
Fixed moving in Blender UI and refresh for relative paths

### PR DESCRIPTION
**Pull request by justo46**
The custom initial transformation matrix is no longer a property, instead obj.matrix_world will be set to it at the beginning.

### CODE REVIEW

File: bseq/operators.py [hunk 1]
```diff
@@ -315,7 +315,24 @@ def execute(self, context):
...
+    '''This changes the timeline start and end frames to the length of a specific sequence'''
+    bl_label = "Set timeline"
+    bl_idname = "bseq.set_start_end_frames"
+    bl_options = {"UNDO"}
+
+    def execute(self, context):
+        scene = context.scene
+        obj = bpy.data.objects[scene.BSEQ.selected_obj_num]
+        (start, end) = obj.BSEQ.start_end_frame
+        scene.frame_start = start
```
Comment (digitalillusions) [inline bseq/operators.py:331]: should start at `0`

File: bseq/operators.py [hunk 2]
```diff
@@ -315,7 +315,24 @@ def execute(self, context):
...
+        scene.frame_end = end
```
Comment (digitalillusions) [inline bseq/operators.py:332]: should end at `end - start`

**Comment by digitalillusions** [comment]
needs some small change, because we use the mod operator for always accessing a valid file `filepath = fs[current_frame % len(fs)]`

**Comment by justo46** [comment]
Should be working now as intended with a button next to Enable/Disable All

**Comment by digitalillusions** [comment]
lgtm

**Pull request merged**